File: 프로그래머스/lv0/120895. 인덱스 바꾸기/인덱스 바꾸기.py
def solution(my_string, num1, num2):
    my_string = list(my_string)
    my_string[num1],my_string[num2] = my_string[num2], my_string[num1]
    return ''.join(my_string)

# list로 변환후 인덱스 바꾸고 붙여서 출력

File: 프로그래머스/lv0/120895. 인덱스 바꾸기/인덱스 바꾸기.java
class Solution {
    public String solution(String my_string, int num1, int num2) {
        String answer = "";
        String str_n2 = "";
        String[] str = my_string.split("");
        int len = my_string.length()-1;
        str_n2 = str[num2];
        str[num2] = str[num1];
        str[num1] = str_n2;
        
        for (int i = 0; i<=len;i++ ){
            answer += str[i];
        }
        return answer;
    }
}